docs/guide/Titanium_SDK/Titanium_SDK_How-tos/User_Interface_Fundamentals/Icons_and_Splash_Screens/README.md

```diff
@@ -31,6 +31,40 @@ You can change but not remove the splash screen that is shown as your app loads.
 
 #### Android splash screen considerations
 
+##### Android 12+
+
+Beginning with Android 12 you will see the icon as a Splashscreen. For more details check [the Andorid guides](https://developer.android.com/develop/ui/views/launch/splash-screen). Make sure you use [Adaptive icons](https://fromzerotoapp.com/android-adaptive-icons/).
```

Comment:
> Make sure you use [Adaptive icons]
Add a reference to the official Google docs as well, e.g. https://developer.android.com/develop/ui/views/launch/icon_design_adaptive

Reply:
:+1: 
added the official link to the "adative icons" and a note about the tutorial